Make.config

```diff
@@ -36,8 +36,7 @@ $(TOP)/Make.config.inc: $(TOP)/Make.config $(TOP)/mk/mono.mk
 
 include $(TOP)/Make.versions
 
-APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/xcode12.5/ab40b131d3e87a96c3414447dcded5fec45bce36/4679032/package/bundle.zip
-
+APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/main/cd2867d44caa9485701b6b63bb6c38f771a767c9/5267313/package/bundle.zip
```

Comment:
Unfortunately this part becomes a bit complicated.

We can't do this, because it will make the api diffs for legacy Xamarin wrong (in fact this should be bumped to the apidiff reference from the `xcode13-ios` branch now).

However, we can't bump to the bundle.zip from the `xcode13-ios` branch, because it won't contain binaries for Xamarin.Mac (nor the .NET assemblies for that matter).

We really need a different bundle.zip for every platform, but for legacy Xamarin that's out of scope for this PR.

But what is possible is a middle ground: use a different url for .NET assemblies, and even better would be to make it possible to use a different url for each platform, so something like this:

```make
APIDIFF_REFERENCES_DOTNET_iOS=...
APIDIFF_REFERENCES_DOTNET_tvOS=...
APIDIFF_REFERENCES_DOTNET_macOS=...
APIDIFF_REFERENCES_DOTNET_MacCatalyst=...
```

because it's entirely possible we'll end up in the same situation again at some point (different stable versions of iOS and macOS), and this way we'd have a solution since we could have different reference bundles for different platforms.